.eslintcache

```diff
@@ -1 +1 @@
\ No newline at end of file
+[{"/Users/Barna/Documents/Code/household-account/src/index.jsx":"1","/Users/Barna/Documents/Code/household-account/src/reportWebVitals.js":"2","/Users/Barna/Documents/Code/household-account/src/App.jsx":"3","/Users/Barna/Documents/Code/household-account/src/TopNav.jsx":"4","/Users/Barna/Documents/Code/household-account/src/Account.jsx":"5","/Users/Barna/Documents/Code/household-account/src/data.js":"6"},{"size":501,"mtime":1607885450289,"results":"7","hashOfConfig":"8"},{"size":362,"mtime":1607885083550,"results":"9","hashOfConfig":"8"},{"size":222,"mtime":1608108459571,"results":"10","hashOfConfig":"8"},{"size":730,"mtime":1608107242171,"results":"11","hashOfConfig":"8"},{"size":5472,"mtime":1608201234411,"results":"12","hashOfConfig":"8"},{"size":723,"mtime":1608200789759,"results":"13","hashOfConfig":"8"},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},"1asedt4",{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/Barna/Documents/Code/household-account/src/index.jsx",[],["27","28"],"/Users/Barna/Documents/Code/household-account/src/reportWebVitals.js",[],"/Users/Barna/Documents/Code/household-account/src/App.jsx",[],"/Users/Barna/Documents/Code/household-account/src/TopNav.jsx",[],"/Users/Barna/Documents/Code/household-account/src/Account.jsx",[],"/Users/Barna/Documents/Code/household-account/src/data.js",[],{"ruleId":"29","replacedBy":"30"},{"ruleId":"31","replacedBy":"32"},"no-native-reassign",["33"],"no-negated-in-lhs",["34"],"no-global-assign","no-unsafe-negation"]
```

Comment:
ezt a file-t nem kell bele tenni a gitbe.

ha meg nincs `.gitignore` (. "." is kell a nevbe), akkor hozd letre. Soronkent 1 dologt lehet bele tenni amit ki akarsz hagyni a gitbol. 

Ez legyen benne:
`.eslintcache`